dashboard/test/helpers/curriculum_helper_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'test_helper'
+
+class CurriculumHelperTest < ActiveSupport::TestCase
+  setup_all do
+    @errors = Object.new
+    @errors.stubs(:add)
+    @object_to_validate = Object.new
+    @object_to_validate.extend(CurriculumHelper)
+    @object_to_validate.stubs(:errors).returns(@errors)
```

Comment:
I love this strategy! this seems way more transparent vs what I had suggested 👍

Reply:
Shoutout to @bethanyaconnor for the help with that one!